app/client/templates/views/account.js

```diff
@@ -4,11 +4,82 @@ Template Controllers
 @module Templates
 */
 
+/**
+Watches custom events
+
+@param {Object} newDocument  the account object with .jsonInterface
+*/
+var addLogWatching = function(newDocument){
+    var contractInstance = web3.eth.contract(newDocument.jsonInterface).at(newDocument.address);
```

Comment:
There was an older geth version, which didn't filter reverted logs and therefore one received the ones from other contracts as well. can you try with the latest develop?